common/djangoapps/util/tests/test_submit_feedback.py

```diff
@@ -44,17 +46,18 @@ def setUp(self):
             username="test",
             profile__name="Test User"
         )
-        # This contains issue_type and course_id to ensure that tags are submitted correctly
         self._anon_fields = {
             "email": "test@edx.org",
             "name": "Test User",
             "subject": "a subject",
             "details": "some details",
+            "issue_type": "test_issue"
+        }
+        self._auth_fields = {
+            "subject": "a subject",
+            "details": "some details",
             "issue_type": "test_issue",
-            "course_id": "test_course"
         }
-        # This does not contain issue_type nor course_id to ensure that they are optional
```

Comment:
Based on the code in `lms/templates/help_modal.html`, it seems like issue_type should always be present.